RecoTracker/LSTCore/src/alpaka/LST.dev.cc

```diff
@@ -104,12 +103,12 @@ void ALPAKA_ACCELERATOR_NAMESPACE::lst::LST::prepareInput(std::vector<float> con
       int pixtype = -1;
```

Comment:
```suggestion
      PixelType pixtype = kInvalid;
```
perhaps this is more clear ... and why not use `PixelType` in the vector types as well; otherwise an enum seems impractical if every other part of the code using these values needs to go back and forth between int, int8_t, PixelType and instead of enum it may be better to use the named constants